Update Sentinel scenario (#73)
hashicorp/katakoda: terraform-sentinel/intro.md

> [Sentinel](https://www.hashicorp.com/sentinel/) is the Policy-as-Code product from HashiCorp that enforces automated enforcement of logic-based policy decisions across all HashiCorp products.

Reviewer: ```suggestion
[Sentinel](https://www.hashicorp.com/sentinel/) is the Policy-as-Code product from HashiCorp that enforces  logic-based policy decisions across all HashiCorp products.
```

> [Sentinel](https://www.hashicorp.com/sentinel/) is the Policy-as-Code product from HashiCorp that enforces automated enforcement of logic-based policy decisions across all HashiCorp products.

Reviewer 2: ```suggestion
[Sentinel](https://www.hashicorp.com/sentinel/) is the Policy-as-Code product from HashiCorp that automatically enforces automated logic-based policy decisions across all HashiCorp products.
```

> It allows customers to implement policy-as-code in the same way that Terraform implements infrastructure-as-code.

Reviewer: ```suggestion
It allows users to implement policy-as-code in a similar way to how Terraform implements infrastructure-as-code.
```

> The Sentinel Command Line Interface (CLI) allows you to apply and test Sentinel policies including those that use mocks generated from Terraform Cloud and Terraform Enterprise plans.

Reviewer: ```suggestion
The Sentinel Command Line Interface (CLI) allows you to apply and test Sentinel policies, including those that use mocks generated from Terraform Cloud and Terraform Enterprise plans.
```
I'm bad at commas, though, so 🧂

> The Sentinel Command Line Interface (CLI) allows you to apply and test Sentinel policies including those that use mocks generated from Terraform Cloud and Terraform Enterprise plans.

Reviewer 2: ```suggestion
The Sentinel Command Line Interface (CLI) allows you to apply and test Sentinel policies using mocks from Terraform Cloud and Terraform Enterprise plans.
```

> This **Sentinel & Terraform Cloud** environment provides you a place to create and test policies for your infrastructure in **30 minutes**.

Reviewer 2: ```suggestion
This **Sentinel & Terraform Cloud** environment provides you a place to create and test policies against pre-populated mock data.
```

Reviewer 2: Not sure if this was referring to the current Katacoda environment or to Sentinel embedded in TFC.

> This scenario follows the **Team & Governance** track in the [Terraform Learn guide](https://learn.hashicorp.com/terraform/sentinel/sentinel-policies).

Reviewer 2: We should put this at the end of the guide instead of here. I don't want users to drop and try the guide instead, or think they should follow the guide instead of the scenario.

Reference Materials:

- [Sentinel documentation](https://docs.hashicorp.com/sentinel)

You may want to try the following tutorials:

> - [Sentinel Policies](https://learn.hashicorp.com/vault/security/iam-sentinel)

Reviewer 2: This should also go at the end as a "further steps" type thing.

> <img src="https://s3.amazonaws.com/terramino.hashicorp.fun/HashiCorp.png" alt="HashiCorp Logo"/>

Reviewer 3: This URL is incorrect and gives a 404. Consider linking to a permanent, stable copy of the logo rather than directly to an S3 bucket. (Or skip it altogether.)

> <img src="https://s3.amazonaws.com/terramino.hashicorp.fun/HashiCorp.png" alt="HashiCorp Logo"/>

Reviewer 2: Looks like this is broken, at least in your personal scenario.
